api_migration.md

```diff
@@ -0,0 +1,121 @@
+# Api Migration
+
+New API can be found in [json format here](./api.json)
+
+## /sectors
+/sectors
+/sectors/:id
+
+/sectors/:id/feedback
+/sectors/:id/visits
+/sectors/:id/activities
+
+## /regions
+/regions
+/regions/:id
+`GET` replaces `/regions/{region}/organisations`  ⏱ volunteer app
+
+## /organisations
+/organisations
+/organisations/:id || me
+`GET` replaces /api/users/cb-name 👣 visitor app 
+
+`PUT` replaces `/api/cb/details/update` 👣 visitor app
+
+### /organisations/:id/visit_activities
+`GET` replaces `/api/activities/today || all` 👣 visitor app // filter used for individual day
+
+`POST` replaces `/api/activity/add` 👣 visitor app 
+
+`PUT` replaces `/api/activity/update` 👣 visitor app 
+
+`DELETE` replaces `/api/activity/delete` 👣 visitor app 
+
+### /organisations/:id/feedback
+`GET` replaces `/api/cb/feedback` 👣 visitor app 
+
+`POST` replaces `/api/cb/feedback` 👣 visitor app 
+
+### /organisations/:id/users
+`GET` replaces `/api/users/all` 👣 visitor app   
+`GET` replaces `/api/visitors/filtered` 👣 visitor app   
+`GET` replaces `/api/visitors/all` 👣 visitor app  
+`GET` replaces `/volunteers/organisation/{organisation}` ⏱ volunteer app  
+`GET` replaces `/volunteers/organisation360/{organisation}` ⏱ volunteer app  
+
+### /organisations/:id/visits
+`GET` replaces `/api/users/filtered`  👣 visitor app   
+`GET` replaces `/api/users/chart-all`  👣 visitor app  
+
+### /organisations/:id/volunteers 
+🤔 volunteers here reffers to volunteer logs. In two minds whether this is a better category over `/volunteer_logs`  
```

Comment:
I commented above that I think the naming should be clearer here and that your later suggestion is a better name.

However, we can also have a `/organisations/:id/volunteers` endpoint, but that endpoint should be concerned with users who are volunteers under the given organisation.